routes/ses.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const express = require("express");
+const bodyParser = require("body-parser");
+
+const router = express.Router();
+const jsonParser = bodyParser.json();
+
+
+router.post("/notification", jsonParser, async (req, res) => {
```

Comment:
Does this need to be an `async` handler?

Reply:
It will be. It will make `await` calls to async `DB` functions for updating the `subscribers` table on bounce & complaint notifications.